### PR TITLE
Merge master to dev branch

### PR DESCRIPTION
**Pull request by jannuary**
See #346

### CODE REVIEW

File: src/window.ui.in [hunk 1]
```diff
@@ -76,7 +76,7 @@
             <child type="end">
               <object class="GtkToggleButton" id="selection_toggle">
                 <property name="receives-default">1</property>
-                <property name="icon-name">object-select-symbolic</property>
+                <property name="icon-name">selection-mode-symbolic</property>
```
Comment (xou816) [inline src/window.ui.in:79]: I don't have this icon as a system icon, is that a very new icon? (GTK 4.4.1 on my end)

Comment (jannuary) [reply src/window.ui.in:79]: [It is](https://gitlab.gnome.org/GNOME/adwaita-icon-theme/-/commit/52c34cbbe2c1f037d3f918e3a9a3d18e7610c8b7), yeah. Flatpak should pull it from the runtime though.

Comment (xou816) [reply src/window.ui.in:79]: right, got it, I had been too lazy to build that via Builder :D

**Comment by xou816** [comment]
LGTM, let's move on with the newest gtk and adwaita ;) thanks!!

**Pull request merged**